atest/testdata/keywords/AsyncLib.py

```diff
@@ -0,0 +1,49 @@
+import asyncio
+
+from robot.libraries.BuiltIn import BuiltIn
+
+
+class Hanger:
+
+    def __init__(self) -> None:
+        self.task = None
+        self.ticks = []
+    
+    async def start_async_process(self):
+        while True:
+            self.ticks.append('tick')
+            await asyncio.sleep(0.5)
+
+async def just_wait():
+    await asyncio.sleep(2)
+
+class AsyncLib:
+
+    def __init__(self) -> None:
+        pass
```

Comment:
This `__ini__` is unnecessary and just adds noise. Also, there should be two empty rows between classes/functions on the module level (PEP-8).